allauth/socialaccount/providers/metamask/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+from .views import login_api
+
+
+urlpatterns = [
+    path("metamask/login_api/", login_api, name="metamask_login"),
```

Comment:
Would prefer to not have "_" in URLs. Perhaps also drop _api? Why not just "metamask/login/" ?

Reply:
Latest PR to my branch fixes this.